face-recognition.ipynb

```diff
@@ -214,9 +214,13 @@
     "    \n",
     "def load_metadata(path):\n",
     "    metadata = []\n",
+    "    s='jpg'\n",
+    "    a='jpeg'\n",
     "    for i in os.listdir(path):\n",
     "        for f in os.listdir(os.path.join(path, i)):\n",
-    "            metadata.append(IdentityMetadata(path, i, f))\n",
+    "            #checking file extention. Allowing only '.jpg' and '.jpeg' files.\n",
+    "            if f.endswith(s) or f.endswith(a):\n",
```

Comment:
Thanks for your pull request. Definitely a useful extension to check for file types. But instead of using string operations like `endswith` please use `os.path.splitext` to obtain the file extension. Alternatively, specify a pattern when loading the files:

```python
...
import glob
import os

class IdentityMetadata():
    def __init__(self, name, file):
        # identity name
        self.name = name
        # image file name
        self.file = file

    ...

    def image_path(self):
        return self.file
    
def load_metadata(path):
    metadata = []
    for name in os.listdir(path):
        for f in glob.glob(os.path.join(path, name, '*.jpg')) + glob.glob(os.path.join(path, name, '*.jpeg')):
            metadata.append(IdentityMetadata(name, f))
```

Note that this requires small changes to the `IdentityMetadata` constructor and the `image_path` method. I actually prefer the second approach. WDYT?

Reply:
I think `splitext` would be good option without changing much in source code.

Reply:
I'm not so much concerned about the amount of changes rather than doing the right things. Which solution do you think is better and why?

Reply:
Thanks for investigating. +1 to use `splitext`. Please update your PR and squash your commits into a single commit and I'll merge.

Reply:
Maybe using 

```python
...
ext = os.path.splitext(f)[1]
if ext == '.jpg' or ext == '.jpeg':
...
```

is a bit more readable ...